internal/endorser/endorser.go

```diff
@@ -58,14 +57,14 @@ func loadAndVerifyProvenances(provenanceURIs []string, referenceValues verifier.
 	}
 
 	// load provenances from URIs
-	provenances := make([]slsa.ValidatedProvenance, 0, len(provenanceURIs))
+	provenances := make([]amber.ValidatedProvenance, 0, len(provenanceURIs))
```

Comment:
I intentionally changed this from `amber` to `slsa` because we are deprecating `amber`. Please keep `slsa`.

Reply:
In fact, `slsa.ValidatedProvenance` and `amber.ValidatedProvenance` are the same struct, but are located in two different packages. As discussed, it is better to have it (and the `ProvenanceIR`) in package `internal/common`. 

The reason I did not already put `ValidatedProvenance` in package `internal/common` was to avoid a circular dependency, and I thought we'd remove amber anyway. I can think of two ways to avoid the circular dependency:
- add a package common (or a similar name) in `pkg`, in addition to the one we currently have in `internal/commom`.
- Move more code from `amber` and `slsa` to `internal/common` to avoid the circular dependency. 

Please let me know if you want to discuss more or prefer me to do this refactoring :)

Reply:
Perhaps some of this generic provenance-related structure and functionality could go to `pkg/provenance`, and be made public as well.

Reply:
Yes! :) I'll do this this in the next PR.